README.md

```diff
@@ -33,29 +36,31 @@ The theme can be built using the following commands:
 $ npm install
 $ npm run build
 ```
-The theme will then be built at `/govuk`, ready for use in the local Keycloak dev server.
+The theme will then be built at `/govuk`, ready for use in the local Keycloak server.
 
-### Local Keycloak dev server
-A local Keycloak server for developing the theme is available as a Docker container.
+### Local Keycloak server
+A local Keycloak server for developing and testing the theme is available as a Docker container.
 
 To bring up the server, make sure you’ve installed and started [Docker Community Edition](https://docs.docker.com/engine/installation/), then use the following commands:
 ```bash
 $ npm install
 $ npm run build
 $ docker-compose up
 ```
-The Keycloak server will now be available on <http://localhost:8080>.
+The Keycloak server will now be available on <http://localhost:8080>. You can log into the Administration Console using “**admin**” as both username and password.
 
-To enable the theme in the local Keycloak server, log in (using “**admin**” as both username and password), and follow steps 2 and 3 in [the “Using” section above](#using).
+<img alt="" src="docs/images/keycloak-local.png" width="512" height="300">
```

Comment:
1. Grand, I’m happy to leave as-is.

2. Yeah I was thinking more for the convenience of inspecting changes in GOV.UK stuff between versions if necessary. But you’re right, it’s unlikely to be a big issue.

One thing that might be more fiddly is keeping to date with the original Keycloak theme files as they change between Keycloak versions. Obviously that’ll be fiddly anyway, given that we’re changing them, but as things stand a developer who’s trying to keep our theme up-to-date with the latest version of Keycloak will have to go and download both the latest version of Keycloak, and whatever version our theme was based on, and diff them. That was my thinking when including the original Keycloak theme files in the repo.

3. Yeah a different repo for every theme feels unnecessary at this point — I’m assuming each of our variations are going to be reasonably small changes to the main theme.

If we had different directories for each new theme, would we copy all the theme files into it? I worry that would make the different themes difficult to maintain if and when the theme template files change in future.